stock_orderpoint_generator/models/product.py

```diff
@@ -26,53 +27,23 @@ def create(self, vals):
             record.auto_orderpoint_template_ids.create_orderpoints(record)
         return record
 
-    @api.multi
     def write(self, vals):
         result = super().write(vals)
         if vals.get("auto_orderpoint_template_ids"):
             orderpoint_templates = self.mapped("auto_orderpoint_template_ids")
             orderpoint_templates.create_orderpoints(self)
         return result
 
-    def _compute_historic_quantities_dict(
-        self, location_id=False, from_date=False, to_date=False
-    ):
-        """Returns a dict of products with a dict of historic moves as for
-           a list of historic stock values resulting from those moves. If
-           a location_id is passed, we can restrict it to such location"""
-        location = location_id and location_id.id
-        domain_quant_loc, domain_move_in_loc, domain_move_out_loc = self.with_context(
-            location=location
-        )._get_domain_locations()
-        if not to_date:
-            to_date = fields.Datetime.now()
-        domain_move_in = domain_move_out = [
-            ("product_id", "in", self.ids),
-            ("state", "=", "done"),
-        ] + domain_move_in_loc
-        domain_move_out = [
-            ("product_id", "in", self.ids),
-            ("state", "=", "done"),
-        ] + domain_move_out_loc
+    def _get_stock_move_domain(self, domain_move=False, from_date=False, to_date=False):
+        domain = [("product_id", "in", self.ids), ("state", "=", "done")] + domain_move
         if from_date:
-            domain_move_in += [("date", ">=", from_date)]
-            domain_move_out += [("date", ">=", from_date)]
-        domain_move_in += [("date", "<=", to_date)]
-        domain_move_out += [("date", "<=", to_date)]
-        move_obj = self.env["stock.move"]
-        # Positive moves
-        moves_in = move_obj.search_read(
-            domain_move_in, ["product_id", "product_qty", "date"], order="date asc"
-        )
-        # We'll convert to negative these quantities to operate with them
-        # to obtain the stock snapshot in every moment
-        moves_out = move_obj.search_read(
-            domain_move_out, ["product_id", "product_qty", "date"], order="date asc"
-        )
-        for move in moves_out:
-            move["product_qty"] *= -1
-        # Merge both results and group them by product id as key
-        moves = moves_in + moves_out
+            domain += [("date", ">=", from_date)]
+        domain += [("date", "<=", to_date)]
```

Comment:
Same here as above. If this will always be used, why is the default value `False` and never checked?